arc42/src/07_deployment_view.adoc

```diff
@@ -248,6 +248,13 @@ The CLI can echo the full schema instead of doing the actual migration, too. Thi
 $ java -jar sandbox-{acc-version}.jar create-schema --spring.datasource.url=$JDBC_URL --spring.datasource.username=$USR --spring.datasource.password=$PASS
 ----
 
+=== Reset Sandbox data
+In a sandbox environment it is a good pattern to reset the data periodically to the factory-defaults. To reset the data of the accelerator, the following steps are needed:
+
+ . Stop the running accelerator process by terminating the jar or by stopping the docker container.
+ . Delete the database data by redeploying your database instance. The procedure of resetting the database depends on your local deployment setup. Check the docs of your preferred database for further information.
+ . Restart the accelerator.
```

Comment:
"docs" can be replaced with "documentation"

Reply:
Done